sdks/java/core/src/test/java/org/apache/beam/sdk/schemas/SchemaTranslationTest.java

```diff
@@ -315,4 +316,36 @@ public void typeInfoNotSet() {
       assertThat(exception.getCause().getMessage(), containsString("TYPEINFO_NOT_SET"));
     }
   }
+
+  /** A simple logical type that has no argument. */
+  public static class NullArgumentLogicalType implements Schema.LogicalType<String, String> {
+    public static final String IDENTIFIER = "beam:logical_type:null_argument:v1";
+
+    public NullArgumentLogicalType() {}
+
+    @Override
+    public String toBaseType(String input) {
+      return input;
+    }
+
+    @Override
+    public String toInputType(String base) {
+      return base;
+    }
+
+    @Override
+    public String getIdentifier() {
+      return IDENTIFIER;
+    }
+
+    @Override
+    public Schema.FieldType getArgumentType() {
```

Comment:
nit: this is in fact nullable
```suggestion
    public @Nullable Schema.FieldType getArgumentType() {
```

Reply:
done.